Tasks/PythonScriptV0/task.json

```diff
@@ -0,0 +1,112 @@
+{
+    "id": "6392F95F-7E76-4A18-B3C7-7F078D2F7700",
+    "name": "PythonScript",
+    "friendlyName": "Python Script",
+    "description": "Run a Python script.",
+    "helpMarkDown": "",
+    "category": "Utility",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 135,
+        "Patch": 0
+    },
+    "preview": true,
+    "demands": [],
+    "instanceNameFormat": "Run Python script",
+    "groups": [
+        {
+            "name": "advanced",
+            "displayName": "Advanced",
+            "isExpanded": false
+        }
+    ],
+    "inputs": [
+        {
+            "name": "targetType",
+            "type": "radio",
+            "label": "Type",
+            "required": true,
+            "defaultValue": "filePath",
+            "helpMarkDown": "Target script type: File path or Inline",
+            "options": {
+                "filePath": "File path",
+                "inline": "Inline"
+            }
+        },
+        {
+            "name": "filePath",
+            "type": "filePath",
+            "label": "Script Path",
+            "visibleRule": "targetType = filePath",
+            "required": true,
+            "defaultValue": "",
+            "helpMarkDown": "Path of the script to execute. Must be a fully qualified path or relative to $(System.DefaultWorkingDirectory)."
+        },
+        {
+            "name": "script",
+            "type": "multiLine",
+            "label": "Script",
+            "visibleRule": "targetType = inline",
+            "required": true,
+            "defaultValue": "",
```

Comment:
You could set the default inline script to the following so that it succeeds faster without modification:

```python
# Write your Python script here.\n\n# Add variables marked secret on the Variables tab to pass secret variables to this script.
```

Reply:
Synced offline, won't fix for now.  We'll hold off on doing secret variables for this task.